charts/pie_test.go

```diff
@@ -0,0 +1,20 @@
+package charts
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPieAssetsBeforeRender(t *testing.T) {
+	bar := NewBar()
+	assert.Equal(t, bar.JSAssets.Values, []string{"echarts.min.js"})
```

Comment:
```suggestion
	assert.Equal(t, []string{"echarts.min.js"}, bar.JSAssets.Values)
```
[func Equal(t TestingT, expected, actual interface{}, msgAndArgs ...interface{}) bool](https://github.com/stretchr/testify/blob/92707c0b2d501c60de82176c4aa1cf880abac720/assert/assertions.go#L334)